homeassistant/components/overkiz/executor.py

```diff
@@ -107,8 +109,8 @@ async def async_execute_command(self, command_name: str, *args: Any) -> None:
             "device_url": self.device.device_url,
             "command_name": command_name,
         }
-
-        await self.coordinator.async_refresh()
+        if refresh_afterwards:
```

Comment:
Can you document in this function why we have `refresh_afterwards`?

Reply:
Regarding this specific thread, maybe my old never merged PR #103672 to execute multiple commands at once will help here?
Please let me know if you want me to refresh it.

Reply:
I find the command queue a much better solution than my workaround. What say you, @iMicknl, will you have time to get back to the command queueing PR?

Reply:
@nyroDev, It might be a good idea to revive this PR #103672. Looks clever solution.

Reply:
Agree, but that PR would need some more love and a bigger refactoring. The current code around commands (and queuing) can be a lot better, but this same list is being used to stop cover movement (cancel execution). Would love to see better command handling, queuing and strong typing here.